test/unit/auth/token-verifier.spec.ts

```diff
@@ -301,13 +291,34 @@ describe('FirebaseTokenVerifier', () => {
               verifyApiName: 'verifyToken()',
               jwtName: 'Important Token',
               shortName: 'token',
-              expiredErrorCode: invalidExpiredErrorCode as any,
-              errorCodeConfig: AUTH_ERROR_CODE_CONFIG,
+              errorCodeConfig: verifierAuth.ID_TOKEN_ERROR_CODE_CONFIG,
+              errorType: invalidErrorCodeTypes as any,
+            },
+            app,
+          );
+        }).to.throw('The provided error type must be a non-null PrefixedFirebaseError type.');
+      });
+    });
+
+    const invalidErrorCodeConfigs = [null, NaN, 0, 1, true, false, [], {}, { a: 1 }, _.noop, '', 'test'];
```

Comment:
These tests now belong in utils.

Reply:
Moved the tests to `test/utils`

Reply:
Let's not move the whole file. It still contains many auth-specific tests.